public/src/components/epicTests/epicTestEditor.tsx

```diff
@@ -121,6 +121,7 @@ interface EpicTestEditorProps extends WithStyles<typeof styles> {
   isArchived: boolean,
   isNew: boolean,
   testNames: string[],
+  testNicknames: string[],
```

Comment:
just wondering why are this and `testNames` plural and an array? everything else in this type seems to be describing an individual test rather than a set of tests, but I'm probably just missing some context to understand this.

Reply:
In order to validate the new name, we have to compare it with the existing names, to make sure that there are no duplicates - so we need to pull in the existing test names and nicknames to compare the new name with.

Reply:
👍